packages/storybook/src/generators/configuration/configuration.spec.ts

```diff
@@ -61,17 +61,8 @@ describe('@nrwl/storybook:configuration', () => {
       );
 
       expect(
-        storybookTsconfigJson.exclude.includes('../**/*.spec.ts')
+        storybookTsconfigJson.exclude.includes('../**/*.spec.{ts,tsx,js,jsx}')
```

Comment:
This one looks like it should only be .ts, the previous tests were expecting .tsx, .js, and .jsx to not be included.

Reply:
Yeah, I know, but I think there's no reason to just be `.ts`, now that I revisited that old code... 😬

Reply:
I think it makes sense to narrow it down to ts/js only if not react at least, maybe something like below suggestion?

Reply:
Yeah ok makes sense!